packages/labs/react/rollup.config.js

```diff
@@ -9,6 +9,6 @@ import {createRequire} from 'module';
 
 export default litProdConfig({
   packageName: createRequire(import.meta.url)('./package.json').name,
-  entryPoints: ['index', 'create-component', 'use-controller'],
+  entryPoints: ['index', 'use-controller'],
   includeNodeBuild: true,
```

Comment:
Just a thought, the node build for this re-exporting package will not look different at all. The same would be true for development builds. Would it be worth the trouble to get rid of that and update the package exports to only have the default and let the source package deal with the conditional exports?

Reply:
There's likely a lot of simplification we could make here, but I don't know if it's worth the effort. I expect most users to migrate pretty quickly since it's trivial to do